torch/csrc/jit/runtime/symbolic_shape_registry.cpp

```diff
@@ -231,12 +231,15 @@ void transformShapeFunction(
   if (schema_string->returns().size() > 1) {
     TORCH_INTERNAL_ASSERT(
         graph->outputs().size() == 1 &&
-        graph->outputs().at(0)->node()->kind() == prim::TupleConstruct);
+        graph->outputs().at(0)->type()->cast<TupleType>());
     auto tuple_node = graph->outputs().at(0)->node();
+    WithInsertPoint guard(graph->return_node());
+    auto tuple_unpack_values = createTupleUnpack(tuple_node->output());
```

Comment:
Should I bother to erase the `TupleConstruct` Operator in the case where `createTupleUnpack` just returns the inputs to the TupleConstruct, and therefore no longer has any uses? I expect DCE to just clean it up anyways. 

```
at::ArrayRef<Value*> createTupleUnpack(Value* v) {
  // small peephole optimization to ensure IntArrayRef attributes can still turn
  // into constants e.g. in x.expand([3, 4])
  if (v->node()->kind() == prim::TupleConstruct) {
    return v->node()->inputs();
  }
  auto& g = *v->owningGraph();
  return g.insertNode(g.createTupleUnpack(v))->outputs();
}
```

Reply:
To me it seems reasonable to leave it as is and let DCE handle it (that's what it was doing previously)